webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -273,6 +329,36 @@ export const ErrataTab = () => {
                 installable: isInstallable,
               } = erratum;
               const isExpanded = erratumIsExpanded(id);
+              let rowActions;
+              if (isInstallable) {
+                rowActions = [
```

Comment:
For non-installable errata, I think these items should still be there, just disabled.

Reply:
Its a different list for non installable but applicable.

Reply:
not sure what you mean here?

Reply:
For applicable errata you want to show `Apply Erratum` which takes em to errata page.   
For installable you want to show `Apply vi Katello agent` etc.

Reply:
Ahhh gotcha.